engine/admin/index.md

```diff
@@ -115,23 +115,18 @@ as the process manager. As of Ubuntu 15.04, Ubuntu uses `systemd` as its process
 manager. For Ubuntu 15.04 and higher, refer to [control and configure Docker
 with systemd](systemd.md).
 
-You configure the `docker` daemon in the `/etc/default/docker` file on your
-system. You do this by specifying values in a `DOCKER_OPTS` variable.
-
 To configure Docker options:
 
 1. Log into your host as a user with `sudo` or `root` privileges.
 
-2. If you don't have one, create the `/etc/default/docker` file on your host. Depending on how
-you installed Docker, you may already have this file.
-
-3.  Open the file with your favorite editor.
+2. Create or edit the `/etc/default/docker` file on your host. If
+   you use `systemd`, create or edit `/etc/systemd/docker` instead.
```

Comment:
Done. I reverted the changes to the Ubuntu install file in favor of the rewrite I did there in #640.